### ISSUE TITLE
Chinese characters show "?"

### ISSUE THREAD

**Issue by rtaromax** [pbugnion/jupyterlab-sql#64]
Hello, when I try to fetch a table with Chinese character in it, it shows "?" instead. How can I fix it? Usually I change system NLS_LANG for this type of problem, but it's not working in this case.

**Comment by pbugnion** [comment]
Good catch. I'm not sure. Could you paste a string here so I can reproduce the issue?
